Volume/Views/WeeklyDebriefView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  WeeklyDebriefView.swift
+//  Volume
+//
+//  Created by Amy Chin Siu Huang on 12/1/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct WeeklyDebriefView: View {
+    @EnvironmentObject private var userData: UserData
+    let weeklyDebrief: WeeklyDebrief
+    @Binding var openedWeeklyDebrief: Bool
```

Comment:
Since we're in the weekly debrief view, it's more concise to call this `isOpen`. Also, make both of these private. You're setting them in `init`.